source/quickstart.txt

```diff
@@ -159,17 +159,19 @@ Follow the instructions below to setup {+c2c-product-name+}.
 Initialize ``mongosync``
 ------------------------
 
-``mongosync`` must create an initial connection to the source and
-destination clusters before it can start to sync data. To create the
-initial connection, issue the following command on a single line (the
-command is reformated here for clarity):
+:program:`mongosync` must create an initial connection to the source and
+destination clusters before it can start to sync data. To create the initial
+connection, issue the following command on a single line (the command is
+reformated here for clarity):
 
 .. code-block:: shell
 
    ./bin/mongosync \
          --cluster0 "mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020" \
          --cluster1 "mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020"
 
+To learn about ``mongosync``, see :ref:`mongosync Reference <c2c-mongosync>`.
```

Comment:
[Suggestion]

Since we've added the link to the mongosync page at the beginning of this section, I don't think we also need a link here at the end.

Reply:
Fixed